src/QsCompiler/Tests.Compiler/CommandLineTests.fs

```diff
@@ -348,7 +348,7 @@ let ``find path absolute to outputfolder with spaces`` () =
 let ``find path relative to here with spaces`` () =
     let fullPath = Path.Combine (Path.GetFullPath "alpha","some beta","c","test 03.qs")
     let options = new BuildCompilation.BuildOptions()
-    let id = CompilationUnitManager.TryGetFileId (new Uri(fullPath)) |> snd
+    let id = CompilationUnitManager.GetFileId (new Uri(fullPath))
```

Comment:
Thanks for updating these!

Reply:
I try to actually pay attention to warnings as I see them. :)